### PR TITLE
Initial Schema

### PR DESCRIPTION
**Pull request by casey-chow**

### CODE REVIEW

File: server/migrations/20190107234122_initial.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      table.increments('id').primary();
+      table.text('name').notNullable();
+      table.text('description').notNullable();
+      table.specificType('industries', 'text ARRAY');
+      table.text('status').notNullable();
+      table.text('contact_email').notNullable();
+      table.json('company_links').defaultTo('{}');
+      table.timestamps();
+    }),
+    knex.schema.createTable('users', (table) => {
```
Comment (VinaiRachakonda) [inline server/migrations/20190107234122_initial.js:13]: Missing role key

File: server/migrations/20190107234122_initial.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        .onDelete('CASCADE');
+      table
+        .foreign('company_id')
+        .references('companies.id')
+        .onDelte('CASCADE');
+      table.decimal('market_score');
+      table.decimal('product_score');
+      table.decimal('team_score');
+      table.decimal('fit_score');
+      table.text('comment');
```
Comment (VinaiRachakonda) [inline server/migrations/20190107234122_initial.js:48]: Should we rename this to voteComment to match the schema on the google doc?

**Review by VinaiRachakonda — CHANGES_REQUESTED** [review]
For onboarding purposes can you update the readme to include directions on knex migrations and integration with a local table?

File: server/migrations/20190107234122_initial.js (hunk 1 above)
Comment (casey-chow) [reply server/migrations/20190107234122_initial.js:13]: I renamed it to `permissions` per default for [feathers-permissions](
https://github.com/feathersjs-ecosystem/feathers-permissions), which encompasses roles as well.

File: server/migrations/20190107234122_initial.js (hunk 2 above)
Comment (casey-chow) [reply server/migrations/20190107234122_initial.js:48]: I can update the google doc—I think calling it `voteComment` is redundant.

**Comment by casey-chow** [comment]
@VinaiRachakonda PTAL

File: README.md [hunk 3]
```diff
@@ -37,6 +39,15 @@ $ feathers help                           # Show all commands
...
 ## Components
 
+### Database
+
+We use [objection.js](http://vincit.github.io/objection.js/) as our ORM, 
+which builds on knex.js. Migrations are handled by knex.js as a result.
+
+[Objection.js Guide](http://vincit.github.io/objection.js/)  
+[Knex.js Guide](https://knexjs.org/#Migrations)  
+[Knex.js Migrations Guide](https://knexjs.org/#Migrations)
```
Comment (VinaiRachakonda) [inline README.md:49]: These are the same link

**Review by VinaiRachakonda — APPROVED** [review]

**Pull request merged**